### PR TITLE
Add Network Info section

### PR DESCRIPTION
**Pull request by beaufortfrancois**

### CODE REVIEW

File: src/content/en/fundamentals/media/fast-playback-with-video-preload.md [hunk 1]
```diff
@@ -364,6 +364,25 @@ users.
 Learn more by reading our complete [Delivering Fast and Light
 Applications with Save-Data] article.
 
+#### Network information
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/media/fast-playback-with-video-preload.md]: Is there a better title that is more usecase led? Intelligent loading based on network information?

Comment (beaufortfrancois) [reply src/content/en/fundamentals/media/fast-playback-with-video-preload.md]: I like it! Thanks!

Comment (beaufortfrancois) [reply src/content/en/fundamentals/media/fast-playback-with-video-preload.md]: Addressed at https://github.com/google/WebFundamentals/pull/5058/commits/9854ec43241cb0773111b1cc05bc6b0c28d0ca19

**Comment by beaufortfrancois** [comment]
@petele @jpmedley I'm good to go!

**Pull request merged**